test/test_cli.py

```diff
@@ -0,0 +1,102 @@
+"""Test for the CLI
+
+"""
+
```

Comment:
You should probably include

```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```
here. Just to be sure that we do not introduce any problematic behaviours without realising it.